docs/docs/working-with-images-in-markdown.md

```diff
@@ -0,0 +1,174 @@
+---
+title: Working With Images in Markdown
+---
+
+When building Gatsby sites composed primarily of markdown pages, insertion of images can enhance the content around it. Adding images can be done in multiple ways.
+
+## Featured images with Frontmatter fields
+
+In sites like a blog, you may want to include a featured image that appears at the top of a page. One way to do this can be by grabbing the image filename from a frontmatter field and then transforming it with `gatsby-plugin-sharp` in a GraphQL query.
+
+It is assumed you already have programatically generated pages from markdown using renderers like `gatsby-transformer-remark` or `gatsby-plugin-mdx`. If not, take a read through up to [Part 7 of the Gatsby Tutorial](tutorial/part-seven/). This will build upon the tutorial and as such, `gatsby-transformer-remark` will be used for this example
+
+> Note: This can be done similarly using [MDX](/docs/mdx/) as well. Instead of the `markdownRemark` nodes in GraphQL, `Mdx` can be swapped in and should work.
+
+To start out, download the plugins for Gatsby-image as mentioned in [Using gatsby-image](/docs/using-gatsby-image/).
+
+```shell
+npm install --save gatsby-image gatsby-transformer-sharp gatsby-plugin-sharp
+```
+
+You will also want to have `gatsby-source-filesystem` installed. Then, configure the various plugins in the `gatsby-config` file.
+
+```js:title=gatsby-config.js
+module.exports = {
+  plugins: [
+    `gatsby-plugin-sharp`,
+    `gatsby-transformer-sharp`,
+    `gatsby-transformer-remark`,
+    {
+      resolve: `gatsby-source-filesystem`,
+      options: {
+        path: `${__dirname}/src/pages`,
+      },
+    },
+  ],
+}
+```
+
+Then, in an example markdown file, add a field called `featuredImage`:
+
+```md:title=src/pages/example-post.md
+---
+title: My Amazing Post
+featuredImage: ./awesome-image.png
+---
+
+Content goes here!
+```
+
+The field is going to be the relative path to the image we want to use.
+
+Next, you can now query it in GraphQL. If the filepath points to an actual image, it will be transformed into a `File` node in GraphQL and then you can get the image data out of it by using the `childImageSharp` field.
+
+This can be added to the page query in the markdown template file. In this example, a [Fluid query](/packages/gatsby-image/#fluid-queries) is used to make a responsive image.
```

Comment:
```suggestion
This can be added to the GraphQL query in a Markdown template file. In this example, a [Fluid query](/docs/gatsby-image#images-that-stretch-across-a-fluid-container) is used to make a responsive image.
```